addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -51,6 +51,12 @@ $-editor-messages-margin-x: 2%;
             &.oe_structure_small {
                 padding: 15px 0px;
             }
+            &.oe_structure_none{
+                height: 0px;    
+                border-style: none;
+                padding: 0px;
+                margin: 0px;
+            }
```

Comment:
Here you add style which says: "When a `oe_structure` asks to have an editor message displayed (`oe_empty`), remove it if it has `oe_structure_none`"... what would be the difference with removing `oe_empty` in those cases ?